src/imputors/substitute.jl

```diff
@@ -37,8 +37,18 @@ Substitute(; statistic=defaultstats) = Substitute(statistic)
 
 function _impute!(data::AbstractArray{Union{T, Missing}}, imp::Substitute) where T
     mask = .!ismissing.(data)
-    x = imp.statistic(disallowmissing(data[mask]))
-    return Base.replace!(data, missing => x)
+    # Since most summary statistics will require some data, we throw a debug message when
+    # all supplied values are missing
+    if any(mask)
+        x = imp.statistic(disallowmissing(data[mask]))
+        return Base.replace!(data, missing => x)
+    else
+        @debug(
```

Comment:
I think this can be a `@warn` or at least an `@info`
this *is* exceptional though no nesc terrible.
If people want to mute it then they can use LoggingExtras or something

Reply:
```suggestion
        @warn(
```